scripts/bench/tts-bench.py

```
@@ -0,0 +1,650 @@
+
+# --- setup -------------------------------------------------------------------
+
+
+def do_setup(cache: Path, engines: list[str]) -> None:
+    cache.mkdir(parents=True, exist_ok=True)
+    if "piper" in engines:
+        piper_dir = cache / "piper"
+        if not (piper_dir / "piper").exists():
+            tarball = cache / "piper.tar.gz"
+            download(PIPER_TARBALL, tarball)
+            subprocess.run(["tar", "xzf", str(tarball), "-C", str(cache)], check=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)